### ISSUE TITLE
attributes doesn't seems to work

### ISSUE THREAD

**Issue by moebiusmania** [matthewp/haunted#41]
hi, I really appreciate what `haunted` is achieving, but writing a small codepen ( https://codepen.io/moebiusmania/pen/ZVGbRP?editors=1011 ) from an example taken from the readme it seems that the attributes value never reach the main function 😢... I'm missing something ?
tnx

**Comment by matthewp** [comment]
Ah, it looks like we haven't documented this yet.  You have to set a `observedAttributes` property on the function.

https://codepen.io/matthewp/pen/dwoXGy?editors=1011

Will keep this issue open to get it documented.

**Comment by moebiusmania** [comment]
thanks @matthewp , that's awesome!
I think it would be nice in the future to have something like this:

```javascript
customElements.define( 'my-app', component(App, ['name']) )
```

what do you think?

**Comment by matthewp** [comment]
Yeah I thought about that sort of thing before. See https://github.com/matthewp/haunted/issues/6 . Went with this option because it was easier for now. But can definitely revisit in the future.
